lib/datasource/index.spec.ts

```diff
@@ -26,13 +27,19 @@ const mavenDatasource = mocked(datasourceMaven);
 const npmDatasource = mocked(datasourceNpm);
 const packagistDatasource = mocked(datasourcePackagist);
 
+const managerList = fs
+  .readdirSync(__dirname, { withFileTypes: true })
+  .filter((dirent) => dirent.isDirectory() && !dirent.name.startsWith('_'))
+  .map((dirent) => dirent.name)
+  .sort();
```

Comment:
Move to test an make async?

Reply:
Okay, but let's leave it sync for simplicity